arch/platform/jn516x/dev/micromac-radio.c

```diff
@@ -463,11 +467,8 @@ prepare(const void *payload, unsigned short payload_len)
 static int
 send(const void *payload, unsigned short payload_len)
 {
-  if(prepare(payload, payload_len) == 0) {
-    return transmit(payload_len);
-  } else {
-    return RADIO_TX_ERR;
-  }
+  prepare(payload, payload_len);
```

Comment:
why no longer check the return value of `prepare`?

Reply:
not strictly needed. but I can change it back